src/views/NewWikiPage.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="new-wiki-page">
+    <b-container>
+      <h1>Nieuwe wikipagina</h1>
+      <div>
+        <div class="form-group">
+          <label for="title">Titel</label>
+          <input class="form-control" type="text" id="title" name="title" v-model="data.title">
+        </div>
+        <div class="form-group">
+          <label for="content">Tekst</label>
+          <textarea class="form-control w-100" id="content" name="content" v-model="data.content" rows="10"></textarea>
+        </div>
+      </div>
+      <button class="btn btn-primary" @click="save">
+        <span v-if="loading">Opslaan...</span>
+        <span v-else>Opslaan</span>
+      </button>
+      <div v-if="error" class="text-danger">{{ error }}</div>
+    </b-container>
+  </div>
+</template>
+
+<script>
+  import Backend from '../services/backend'
+
+  export default {
+    name: 'NewWikiPage',
+    data() {
+      return {
+        data: {
```

Comment:
it's best not to use `data` as an attribute name, since `data` is also used to refer to the data attributes of the component. Better is to call it something like `newWikiObject`, imo